docs/content/releases.md

```diff
@@ -6,6 +6,51 @@ doctypes: ["concept"]
 toc: true
 docs: "DOCS-616"
 ---
+## 3.4.0
+
+%%RELEASE_DATE%%
+
```

Comment:
The default_server listeners for ports 80 and 443 can now be fully customized giving you the flexibility to shift the HTTP and HTTPS default listeners to other ports as your needs require.

Traffic splits now support weights from 0 - 100 giving you the control that you expect when performing canary roll outs of your back end services.

A new capability of "upstream backup" has been introduced for NGINX Plus customers.  This gives you the control to set a backup service for any path.  This takes advantage of NGINX health checks and will automatically forward traffic to the backup service when all pods of the primary service stop responding.

Dynamic reloading of SSL certificates takes advantage of native NGINX functionality to dynamically load updated certificates when they are requested and thus not require a reload when certificates update.

A number of Helm enhancements have come directly from our community and range from giving greater flexibility for HPA, namespace sharing for custom sidecars, and supporting multiple image pull secrets for greater deployment flexibility.

Reply:
That will probably need a review by another set of eyes to verify statements.

Reply:
@brianehlert how do you feel about linking to specific examples in here? For example with the traffic splitting feature, we can add "Try out our [traffic-splitting](https://github.com/nginxinc/kubernetes-ingress/tree/main/examples/custom-resources/traffic-splitting) example deployment!"

Reply:
@brianehlert Can we add a line at the top to talk about the change to our helm chart location?

Something like this
"To make sure the NGINX Ingress Controller follows Helm best practices, we've refactored our helm chart location. You can now find our helm charts under `charts\nginx-ingress`"

Reply:
@brianehlert do we want to highlight the addition of waf bundle capability this time around?

Reply:
> @brianehlert do we want to highlight the addition of waf bundle capability this time around?

It already released, but we can mention it again since the doc is releasing and NIM Security Monitor released its support.  Great idea.

Reply:
> @brianehlert how do you feel about linking to specific examples in here? For example with the traffic splitting feature, we can add "Try out our [traffic-splitting](https://github.com/nginxinc/kubernetes-ingress/tree/main/examples/custom-resources/traffic-splitting?rgh-link-date=2023-12-15T15%3A18%3A09Z) example deployment!"

I think that would be good.